packages/apps/graph/src/services/tracing/trace-analyser.ts

```diff
@@ -0,0 +1,131 @@
+import { Command, Option } from 'commander';
+import { readFileSync } from 'fs';
+
+new Command()
+  .command('trace-analyser')
+  .description('Analyse the traces.log file')
+  .addOption(
+    new Option(
+      '-l, --limit <number>',
+      'Limit the number of queries to display',
+    ),
+  )
+  .addOption(
+    new Option(
+      '-s, --sort <string>',
+      'Sort the queries by the given property',
+    ).default('median'),
+  )
+  .action(async (args) => {
+    try {
+      readFileSync('traces.log');
```

Comment:
How about passing the filepath as an option. Makes it easier to have multiple tracelogs and ability to compare them

Reply:
Good idea, done!